### PR TITLE
[13.0][IMP] stock_picking_operation_quick_change: Allow to filter by product

### PR DESCRIPTION
**Pull request by pedrobaeza**
You can now also filter the lines to change by product.

@Tecnativa

### CODE REVIEW

**Review by cesar-tecnativa — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@CarlosRoca13 please review

**Review by CarlosRoca13 — APPROVED** [review]

File: stock_picking_operation_quick_change/wizards/stock_picking_wizard.py [hunk 1]
```diff
@@ -94,10 +111,17 @@ def action_apply(self):
...
-            )
+            if self.old_location_dest_id:
+                matched_op = matched_op.filtered(
+                    lambda x: x.location_dest_id == self.old_location_dest_id
+                )
+            # Only write operations destination location if the product is the same
+            if self.product_id:
+                matched_op = matched_op.filtered(
+                    lambda x: x.product_id == self.product_id
+                )
```
Comment (cesar-tecnativa) [inline stock_picking_operation_quick_change/wizards/stock_picking_wizard.py:125]: ¿Could you combine these two if-s in one?

**Review by cesar-tecnativa — CHANGES_REQUESTED** [review]

File: stock_picking_operation_quick_change/wizards/stock_picking_wizard.py (hunk 1 above)
Comment (pedrobaeza) [reply stock_picking_operation_quick_change/wizards/stock_picking_wizard.py:125]: Each `if` has its own `filterered`, so they can't be combined.

Comment (cesar-tecnativa) [reply stock_picking_operation_quick_change/wizards/stock_picking_wizard.py:125]: :+1:

**Pull request merged**